getting-started/index.rst

```diff
@@ -127,6 +127,19 @@ tools installed:
 * `ccache <https://ccache.dev>`_ for faster rebuilds
 * `eatmydata <https://www.flamingspork.com/projects/libeatmydata/>`_ for faster testsuite runs
 
+If you want to be able to build distribution tarballs (with ``make dist``) then
+you'll also need some further tools:
+
+* doxygen (v1.8.8 is used for 1.3.x snapshots and releases; 1.7.6.1 fails to
+  process trunk after ``PL2Weight`` was added).
```

Comment:
We should probably say "git master" not "trunk".

Reply:
Are those version numbers still correct? 1.5.x snapshots surely, for one :)

Reply:
Also, 1.8.8 was released in 2014, and debian oldoldstable is probably the last thing standing with it. (oldstable has 1.8.13, as does Ubuntu 1804 — 1604 has 1.8.11.) We probably don't need to mention 1.7.x, which is now very very old.

Reply:
I've tried not to spend time worrying about removing minimum required version information like this that may have become irrelevant - things like RHEL and mingw (the 32-bit only version rather than mingw64) tend to have really ancient versions of things, and there's no easy way I'm aware of to search for packaged versions across all distros, etc.  E.g. a quick dig suggests [RHEL 6 is still in support](https://en.wikipedia.org/wiki/Red_Hat_Enterprise_Linux#RHEL_6) and [comes with doxygen 1.6.1](http://ftp.redhat.com/redhat/linux/enterprise/6Server/en/os/SRPMS/doxygen-1.6.1-6.el6.src.rpm).  I couldn't easily seem to find what RHEL7 has.

If a minimum requirement is not relevant to the reader it's easy to ignore, but if it is relevant having it clearly specified can save a lot of wasted time, so erring on keeping it seems better.

Such info tends to get updated eventually because some new minimum version requirement comes along.  That's not exactly happened here yet, but this is out of date as `bootstrap` no longer downloads and installs a private build of a particular version of doxygen (since 9fd62725125743a9d9641efe2ebaab3d69ce03d7 on 2016-01-23) so now snapshots use whatever doxygen version thyestes has and releases use whatever my dev box has.

Reply:
👍